src/routes/index.js

```diff
@@ -13,6 +17,9 @@ export default class index extends Component {
         <Route path="/dashboard" exact component={Dashboard} />
         <Route path="/signup" exact component={Signup} />
         <Route path="/verify-email" exact component={Confirm} />
+        <Route path="/forgot-password" exact component={SendResetEmailView}/>
+        <Route path="/password-reset-process" exact component={PasswordResetProcess}/>
```

Comment:
@Moise1  Just a suggestion, can't you call ```/password-reset-process``` something short and clear? Take a look at my suggestion

```suggestion
        <Route path="/password-reset" exact component={PasswordResetProcess}/>
```

Reply:
@AbdoulNuru , yeah! A user cannot change the password just yet. The story that I'm working on has a main action of sending the link to the user.

Reply:
@hakoemmy , I'd like to keep it that way coz the process isn't complete until a user provides a new password.